### DISCUSSION TITLE
Will this work on CircuitPython?

### DISCUSSION THREAD

**Discussion by czei** [rdehuyss/micropython-ota-updater#29] [General]
Sorry if this is a dumb question, but I'm only familiar with CircuitPython and not with the differences between MicroPython and CircuitPython.   

Before I go down a rathole of trying to make this work on CircuitPython it doesn't hurt to ask if anyone has tried it.

**Comment by rdehuyss** [comment]
No idea :-)

**Reply by czei** [reply]
fair enough. I will have to give it a try and report back.
